common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -501,4 +501,15 @@ public static byte[] base64Decode(@NonNull final String input, int flags, @NonNu
             throw e;
         }
     }
+
+    /**
+     * Helper function to overwrite all characters in an array with zero
+     * @param chars
+     */
+    public static void overwriteWithZero(char[] chars) {
+        int length = chars == null ? 0: chars.length;
+        for (int i =0; i < length; i++) {
+            chars[i] = '\0';
```

Comment:
I don't think this is zero as the Javadoc suggests. I believe this is the null character. You may want to update your javadoc

Reply:
updated